developer-scripts/README.md

```diff
@@ -34,58 +33,62 @@ developer-scripts/run.sh
+4. The setup is done now. We need to record the mqtt uuid for this object to be used in the later process. 
+    1. Click **edge** in the left menu. Then click **local_edge** in the node list to enter the EMQX edge dashboard.
+    2. In the edge dashboard, click **clients** in the left menu. There should be one client. Copy the client id and save it for the later use.
 
 ### Setup Kuiper
 
-#### Create a Neuron stream
+Open Kuiper dashboard by click `Kuiper` in the left menu, and then click `local_kuiper` node. Then do the following setup.
 
-TODO..
+1. Create stream for neuron. Click "Create Stream", and setup as below. The last section of the Data Source field is the neuron client id in mqtt broker which we have saved before. Click "Submit".
 
-#### Create a rule 
+   ![Create kuiper stream for neuron](resources/create_stream.png)
+   
+2. Switch to "Plugins" tab, click "Create plugin", and setup as below. This will create the tdengine plugin so that the rule result can be ported to tdengine.
+     
+   ![Create kuiper plugin for tdengine](resources/create_plugin.png)
+   
+3. Create a rule. A rule will be created for subscribing data published by Neuron, and send the analysis result to TDengine. Switch to "Rules" tab, click "Create rule". Switch to "Text mode" in the top-right. Fill in Rule ID: **ruleNeuron** or any rule name you want. Enter the below json in the "text" field.
 
-A rule will be created for subscribing data published by Neuron, and send the analysis result to TDengine. 
-
-```json
-{
-  "sql": "SELECT tele[0]->Tag00001 AS temperature, tele[0]->Tag00002 AS humidity FROM neuron",
-  "actions": [
+    ```json
     {
-      "tdengine": {
-        "ip": "taos",
-        "port": 0,
-        "user": "root",
-        "password": "taosdata",
-        "database": "db",
-        "table": "t",
-        "fields": ["temperature","humidity"],
-        "provideTs": false,
-        "tsFieldName": "ts"
-      }
+      "sql": "SELECT tele[0]->Tag00001 AS temperature, tele[0]->Tag00002 AS humidity FROM neuron",
+      "actions": [
+        {
+          "tdengine": {
+            "ip": "taos",
+            "port": 0,
+            "user": "root",
+            "password": "taosdata",
+            "database": "db",
+            "table": "t",
+            "fields": ["temperature","humidity"],
+            "provideTs": false,
+            "tsFieldName": "ts"
+          }
+        }
+      ]
     }
-  ]
-}
-```
-
-
-
-### Query data in TDengine
-
-TODO...
-
-### Data visualization data with Grafana
-
-TODO...
-
-
+    ```
+4. Click "Submit", make sure the rule is up and running. Click the rule status, there should be data in and out in the metrics.
+5. Query data in TDengine. Install TDengine client per [TAOs documentation](https://www.taosdata.com/en/getting-started/). The rule data is in database **db**, table **t**. Query the data with SQL like `use db;select * from t;`.
```

Comment:
`docker exec -it manager-taos /bin/sh` and enter TDengine docker, it already has the TDengine client shell.